### ISSUE TITLE
Android: SetPlaybackRate "forgotten" after closing/opening media

### ISSUE THREAD

**Issue by artoonie** [RenderHeads/UnityPlugin-AVProVideo#1275]
Note: may be related to #1274 

**Describe the issue**
`SetPlaybackRate` is forgotten after closing/opening media

**Your Setup (please complete the following information):**
 - Unity version: 2021.3.7f1
 - AVPro Video version (number and edition (trial/core/ultra/enterprise)): 2.5.5 Core
 - Operating system version: Android 11
 - Device model: Pixel 2
 - Video specs (resolution, frame-rate, codec, file size): 1080, h264, 600mb

**To Reproduce**
1. Open a video in `MediaPlayer` (have not tested with `ExoPlayer`)
2. Change Playback Rate to 4x speed
3. CloseMedia, OpenMedia
4. Note that `MediaPlayer.PlaybackRate` is still 4, but the video plays back at 1x

**Workaround**
It looks like under the hood, nothing happens if PlaybackRate is set to the same value, so you have to change it "away" then back:
```
float playbackRate = m_MediaPlayer.PlaybackRate;
m_MediaPlayer.PlaybackRate = 1;
m_MediaPlayer.PlaybackRate = playbackRate;
```
I call this on `MediaPlayerEvent.EventType.Started`

(side note: Not sure if this happens with ExoPlayer too. It doesn't work out of the box; I get `AndroidJavaException: java.lang.NoClassDefFoundError: com.google.android.exoplr2avp.audio.AudioCapabilities` and haven't investigated further.)

**Comment by Ste-RH** [comment]
A fix has gone in for this and will be available in the next version (v2.6.1)

**Comment by Chris-RH** [comment]
Version 2.6.1 has been released. Please let us know if the issue has not been fixed for you.
